src/Components/Timer.tsx

```diff
@@ -6,27 +6,46 @@ interface Inputs {
   refreshInterval: number;
   refreshIntervalUnits: string;
   timerRunning: boolean;
+  resetFn(): void;
+  setTimerRunning(x): void;
 }
 
 export function TimerUI(props: Inputs) {
+  function determineInitialTime() {
+    var time = ms(String(props.refreshInterval) + props.refreshIntervalUnits);
+    if (time > 1000 * 60 * 60 * 24) {

```

Comment:
Greater than a day?  Why would this matter?

Reply:
It was a timer reset issue but I know how to fix it